tests/test_operation.py

```diff
@@ -477,6 +523,38 @@ def circuit():
             circuit()
 
 
+class TestOperationIntegration:
+    """ Integration tests for the Operation class"""
+
+    def test_inverse_of_operation(self):
+        """Test that the inverse of an operation is """
+
+        dev1 = qml.device("default.qubit", wires=2)
+
+        @qml.qnode(dev1)
+        def circuit():
+            qml.PauliZ(wires=[0], do_queue=True)
```

Comment:
Why specify `do_queue=True` in these two lines?

Reply:
My mistake, correcting it, thanks!